### DISCUSSION TITLE
How to fix "import.meta.url" is not available

### DISCUSSION THREAD

**Discussion by HendrikJan** [nuxt/nuxt#17898] [Questions] [answered]
When using nuxt-bridge, `__dirname` is not available.

I use the alternative: `new URL('.', import.meta.url)` but get this error:

```
"import.meta" is not available in the configured target environment ("es2019") and will be empty
```

How can I set nuxt-bridge to use another target?
Which target should I use?

--------
I found that I can set the nitro target in `nuxt.config.js`:
```javascript
{
    nitro: {
        esbuild: {
            options() {
                return {
                    target: 'es2020',
                };
            },
        },
    },
}
```

but this does not remove the error.

**Comment by benoitlahoz** [comment, marked as answer]
Same here with nuxt 3... Did you manage to fix this error?

**UPDATE**
It worked with: 

```
{
    nitro: {
        esbuild: {
            options: {
                target: 'es2020',
            },
        },
    },
}
```

**Comment by HendrikJan** [comment]
I marked @benoitlahoz his observation as an answer, because it seems like the easiest.

But I also found another.
My problem was also fixed by renaming my `.js` files to `.mjs` files.
